active-directory-wpf-msgraph-v2/MainWindow.xaml.cs

```diff
@@ -47,7 +42,13 @@ private async void CallGraphButton_Click(object sender, RoutedEventArgs e)
             // WAM will always get an account in the cache. So if we want
             // to have a chance to select the accounts interactively, we need to
             // force the non-account
-            if (howToSignIn.SelectedIndex == 1) // Account signed-in with Windows
+            if (howToSignIn.SelectedIndex == 0) // Account signed-in with Windows
+            {
+                firstAccount = PublicClientApplication.OperatingSystemAccount;
+            }
+
+            // WAM
+            else if (howToSignIn.SelectedIndex == 1)
```

Comment:
Please add a comment like the one at line 45, explaining what index 1 and 2 are (i.e. what label it corresponds to)